CONTRIBUTING.rst

```diff
@@ -35,37 +43,47 @@ No local Postgres server needed.
 
 In the virtual environment you need to install *aiopg* itself and some
 additional development tools (the development tools are needed for running
-the test suite and other development tasks)::
+the test suite and other development tasks)
+
+.. code-block:: shell
 
-   $ pip install -Ue .
-   $ pip install -Ur requirements.txt
+    $ pip install -Ue .
+    $ pip install -Ur requirements.txt
 
 That's all.
 
-To run all of the *aiopg* tests do::
+To run all of the *aiopg* tests do
```

Comment:
You still want a single colon here:
```suggestion
To run all of the *aiopg* tests do:
```
Double colon creates a block and is rendered as a single colon, single colon just does nothing and is rendered as is.